ingest_schemas/templates/benthic_photo_quadrat_transect_template.csv

```diff
@@ -0,0 +1 @@
+Site *,Management *,Sample date: Year *,Sample date: Month *,Sample date: Day *,Sample time,Depth *,Transect number *,Transect label,Transect length surveyed *,Number of quadrats *,Quadrat size *,Number of points per quadrat *,Visibility,Current,Relative depth,Tide,Notes,Observer emails *,Quadrat *,Benthic attribute *,Growth form,Number of points *
```

Comment:
What about reef slope? Also, should we align column order with that of PIT?

Reply:
There's no reef slope in the mockup that Em gave us.  I setup the csv columns to align with the form